index.html

```diff
@@ -38,39 +46,48 @@ <h2>I am a Software Developer</h2>
   </section>
 
   <section class="works-section">
-    <h2>My Recent Works</h2>
-    <hr>
+    <div id="works-header">
+      <h2>My Recent Works</h2>
+      <hr>
+    </div>
     <div class="cards-container">
       <div class="card">
-        <img src="./assets/img/project1-img.png" alt="Multi-Post Stories Project">
-        <h2>Multi-Post Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or sign-ups required. 
-          has been the industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
-        <ul>
-          <li>css</li>
-          <li>html</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">
-          See Project
-        </button>
+        <div class="card-img">
+          <img src="./assets/img/project1-img.png" alt="Multi-Post Stories Project">
+        </div>
+        <div id="leading-card">
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="see-leading-project">
+            See Project
+          </button>
+
+        </div>
       </div>
-      <div class="card card-with-bgimage">
-        <h2>Profesional Art Printing Data</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or sign-ups required. 
-          has been the industry's standard
-        </p>
-        <ul>
-          <li>html</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button type="button">
+      <div class="card card-with-bgimage active">
+        <div class="hide">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button type="button" id="show" class="see-project">
           See Project
         </button>
       </div>
```

Comment:
### OPTIONAL
- [ ] just as the[ first reviewer](https://github.com/emmiiorji/my_new_portfolio/pull/4/files#r840678187) requested, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears at the bottom of the image
- [ ] to achieve this, you can add a display none to your button  to hide button when the page initially loads and for the description and dark opacity a display none on hover, then finally on hover you add a display block to your button, kindly research more about [this](https://www.w3schools.com/howto/howto_css_display_element_hover.asp), all cards on hover should look like your first card 👍🏿 
- Adding this hover effect to your project will be a cool feature 😉

Reply:
Thank you @prolajumokeoni.

I had already done this. The screenshot shown in the comment was taken with the mouse hovering on one of the cards (the one that has the "See Project" button showing).

Reply:
Hi @prolajumokeoni

I'm sorry. I just discovered I didn't push the code from my local repo before requesting another review.